### PR TITLE
feat: Download template in massmint onboard

### LINKED ISSUE #7920: Download template in massmint onboard
discussed with @JustLuuuu 

- adding a download template to the code structure preview could be beneficial. 
- design needed
- I guess I will also need help from someone from @kodadot/internal-dev to write some comments into the comments and placeholders

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Feature

  ## Needs Design check

  - @exezbcz please review

  ## Context

  - [x] Closes #7920
  - [ ] Requires deployment <snek/rubick/worker>

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [x] My fix has changed UI

image

  ## Copilot Summary
  
### <samp>🤖 Generated by Copilot at 943c05f</samp>

Added links, buttons, and files to improve the onboarding page for mass minting. The files provide template formats and examples for csv, json, and txt files that users can use for mass minting.

  
### <samp>🤖 Generated by Copilot at 943c05f</samp>

> _Mass minting on board_
> _Download `csv`, `json`, `txt`_
> _Winter tutorial_

### CODE REVIEW

**Comment by vikiival** [comment]
Maybe `float: left`, underlined and some more visible color?

WdyT @exezbcz ?

**Comment by Jarsen136** [comment]
> @Jarsen136 I would unify all templates according to `template.txt`

Will update it

File: components/massmint/OnBoarding.vue [hunk 1]
```diff
@@ -1,10 +1,15 @@
 <template>
   <div>
     <div class="is-flex is-align-items-center mb-7 px-8">
-      <div
-        class="is-size-2-desktop is-size-2-tablet is-size-3-mobile is-flex is-flex-grow-1 is-justify-content-center has-text-weight-bold">
+      <a
```
Comment (roiLeo) [inline components/massmint/OnBoarding.vue:4]: `nuxt-link :to=`?

File: components/massmint/OnBoarding.vue [hunk 2]
```diff
@@ -51,6 +56,17 @@
...
               class="fixed-height white-space-break-spaces-mobile code" />
+            <div class="is-flex is-justify-content-flex-end mt-2">
+              <NeoButton
+                v-safe-href="
+                  `/massmint/template.${activeDescriptionTab.toLowerCase()}`
+                "
+                :label="$t('massmint.onboarding.downloadTemplate')"
+                variant="text"
+                no-shadow
+                tag="a"
```
Comment (roiLeo) [inline components/massmint/OnBoarding.vue:67]: `nuxt-link`

File: components/massmint/OnBoarding.vue [hunk 3]
```diff
@@ -51,6 +56,17 @@
                 descriptionTabs[activeDescriptionTab].fileStructureDescription
               "
               class="fixed-height white-space-break-spaces-mobile code" />
+            <div class="is-flex is-justify-content-flex-end mt-2">
+              <NeoButton
+                v-safe-href="
```
Comment (roiLeo) [inline components/massmint/OnBoarding.vue:61]: `:to`

File: locales/en.json [hunk 4]
```diff
@@ -1279,6 +1279,7 @@
       "skip": "Skip",
       "next": "Next",
       "done": "Done",
+      "downloadTemplate": "Download a file template",
```
Comment (roiLeo) [inline locales/en.json:1282]: better?
```suggestion
      "downloadTemplate": "Download file template",
```

**Review by roiLeo — APPROVED** [review]
otherwise lg

File: locales/en.json (hunk 4 above)
Comment (Jarsen136) [reply locales/en.json]: Updated. it's better

File: components/massmint/OnBoarding.vue (hunk 1 above)
Comment (Jarsen136) [reply components/massmint/OnBoarding.vue]: Updated

File: components/massmint/OnBoarding.vue (hunk 2 above)
Comment (Jarsen136) [reply components/massmint/OnBoarding.vue:65]: `<a/>` tag with the attribute `download` could make the download functionality work.

File: components/massmint/OnBoarding.vue (hunk 3 above)
Comment (Jarsen136) [reply components/massmint/OnBoarding.vue:59]: <a/> tag with the attribute download could make the download functionality work.

Comment (roiLeo) [reply components/massmint/OnBoarding.vue:59]: oh, I see

**Review by roiLeo — APPROVED** [review]

**Review by daiagi — APPROVED** [review]

**Comment by exezbcz** [comment]
Icon and underlined text or blue text could be enough. Thank you!

**Comment by Jarsen136** [comment]
> Icon and underlined text or blue text could be enough. Thank you!

✅ Updated

image

**Comment by vikiival** [comment]
pay 25 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $25 @ 4.5 USD/DOT ~ 5.556 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x7017ef6ff017d081b8a0d28592d5248416f7821ea1b32a00b830efcbd0e23ca8](https://polkadot.subscan.io/extrinsic/0x7017ef6ff017d081b8a0d28592d5248416f7821ea1b32a00b830efcbd0e23ca8)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**